packages/modules/src/modules/banners/common/choiceCard/ChoiceCardInteractive.tsx

```diff
@@ -213,42 +170,34 @@ export const ChoiceCardInteractive: ReactComponent<ChoiceCardInteractiveProps> =
         // Something is wrong with the data
         if (!Array.isArray(requiredAmounts) || !requiredAmounts.length) {
             return (
-                <div css={style.amountsOrOtherButton}>
-                    <ChoiceCard
-                        value="third"
-                        label="Other"
-                        id="contributions-banner-third"
-                        checked={true}
-                        cssOverrides={style.buttonOverride}
-                    />
-                </div>
+                <ChoiceCard
+                    value="third"
+                    label="Other"
+                    id="contributions-banner-third"
+                    checked={true}
+                    cssOverrides={style.buttonOverride}
+                />
             );
         }
 
-        return (
-            <>
-                <div css={style.amountsButton}>
-                    <ChoiceCardAmount amount={requiredAmounts[0]} />
-                    <ChoiceCardAmount amount={requiredAmounts[1]} />
-                </div>
-                {hideChooseYourAmount ? (
-                    <div css={style.amountsOrOtherButton}>
-                        <ChoiceCardAmount amount={requiredAmounts[2]} />
-                    </div>
-                ) : (
-                    <div css={style.amountsOrOtherButton}>
-                        <ChoiceCard
-                            value="other"
-                            label="Other"
-                            id="contributions-banner-other"
-                            checked={selection.amount === 'other'}
-                            onChange={() => updateAmount('other')}
-                            cssOverrides={style.buttonOverride}
-                        />
-                    </div>
-                )}
-            </>
-        );
+        return [
```

Comment:
Return an array instead of a fragment, as well as using a function instead of a component in order to have the children be correctly cloned by the source group component
https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-react-components/src/choice-card/ChoiceCardGroup.tsx#L120